packages/dd-trace/src/appsec/reporter.js

```diff
@@ -48,6 +47,7 @@ function resolveHTTPResponse (context) {
   const headers = context.resolve(addresses.HTTP_INCOMING_RESPONSE_HEADERS)
 
   return {
+    endpoint: context.resolve(addresses.HTTP_INCOMING_ENDPOINT),
```

Comment:
Wouldn't that already be part of the trace?

Reply:
The tracer already adds `http.route` but the AppSec over Spans RFC specifies that I need to add `http.endpoint`. Go figure...

Reply:
Most tags are duplicated by AppSec at this point. We should try to normalize these and stop duplicating tags every time an existing value is needed.